docs/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -18,6 +18,7 @@ notSupported:
   - javascript.nestjs
   - javascript.cloudflare
 description: "Configuring Session Replay to maintain user and data privacy."
+customCanonicalTag: "/platforms/javascript/session-replay/privacy/"
```

Comment:
Looks like we're inconsistent between camelCase and snake_case naming for our frontmatter properties, alas. Maybe a cleanup task someday